packages/babel-helper-create-class-features-plugin/src/misc.js

```diff
@@ -86,10 +83,17 @@ export function extractComputedKeys(ref, path, computedPaths, file) {
   const declarations = [];
 
   for (const computedPath of computedPaths) {
-    computedPath.traverse(classFieldDefinitionEvaluationTDZVisitor, {
-      classBinding: path.node.id && path.scope.getBinding(path.node.id.name),
-      file,
-    });
+    if (computedPath.get("key").isReferencedIdentifier()) {
+      classFieldDefinitionEvaluationTDZVisitor.ReferencedIdentifier.call(
+        computedPath.get("key"),
+        {
+          classBinding:
+            path.node.id && path.scope.getBinding(path.node.id.name),
+          file,
+        },
+      );
+    }
+    computedPath.get("key").traverse(classFieldDefinitionEvaluationTDZVisitor);
```

Comment:
Here you should still pass the state object (the second parameter)

Reply:
Done. PTAL.